src/Features/Core/Portable/ChangeSignature/DelegateInvokeMethodReferenceFinder.cs

```diff
@@ -104,8 +105,21 @@ protected override bool CanFind(IMethodSymbol symbol)
                 .Where(e => semanticModel.GetSymbolInfo(e, cancellationToken).Symbol.OriginalDefinition == methodSymbol);
 
             return invocations.Concat(convertedAnonymousFunctions).SelectAsArray(
-                n => new FinderLocation(n, new ReferenceLocation(document, null, n.GetLocation(), isImplicit: false,
-                    symbolUsageInfo: GetSymbolUsageInfo(n, semanticModel, syntaxFactsService, semanticFactsService, cancellationToken), candidateReason: CandidateReason.None)));
+                  n => new FinderLocation(
```

Comment:
Nit: While you are touching this, can we rename `n` to something more meaningful?